README.md

```diff
@@ -97,16 +122,20 @@ Currently the tool supports a single subcommand, `git dropbox set-head <remote>
   it's a good idea to use [selective
   sync](https://help.dropbox.com/installs-integrations/sync-uploads/selective-sync-overview)
   and disable syncing of the folder containing the repository to avoid any
-  unexpected conflicts, just in case.
+  unexpected conflicts, just in case
+
 - git-remote-dropbox does not use the Dropbox desktop client - it uses the API
-  directly. It does not require that the desktop client is installed.
-- The remote helper does not support shallow cloning.
+  directly. It does not require that the desktop client is installed
+
+- The remote helper does not support shallow cloning
+
 - Cloning a repository or fetching a lot of objects produces lots of loose
-  objects. To save space in the local repository, run `git gc --aggressive`.
+  objects. To save space in the local repository, run `git gc --aggressive`
+
 - If the remote HEAD (default branch on the remote) is not set, after cloning a
   repository from Dropbox, Git will not automatically check out a branch. To
   check out a branch, run `git checkout <branch>`. To set the default branch on
-  the remote, use the [git-dropbox-manage](#repository-manager) command.
+  the remote, use the [git-dropbox-manage](#repository-manager) command
```

Comment:
Why no periods at the end of sentences? Is it because they are bullet points? If there are multiple / complete sentences in bullets, I'd like to keep the trailing periods.

Reply:
My preference, but your repo, your style, your rules!